bedrock/products/templates/products/vpn/landing.html

```diff
@@ -81,7 +81,7 @@
               </a>
 
               <p class="availability-copy">
-                {{ ftl('vpn-shared-available-countries-v4', fallback='vpn-shared-available-countries-v3') }}
+                {{ ftl('vpn-shared-available-countries-v5', fallback='vpn-shared-available-countries-v4') }}
```

Comment:
**issue (blocking):** I can see waitlist copy and Finland in available countries list on http://localhost:8000/en-US/products/vpn/?geo=fi (same with ?geo=se & Sweden)
<img width="614" alt="Screen Shot 2022-02-21 at 11 56 26 AM" src="https://user-images.githubusercontent.com/19650432/154951883-521a7e6b-5684-42ab-aca9-efb607774646.png">


**suggestion:** move this string behind the switch as well

Reply:
Ah, nice catch!